Source/WPEFramework/PluginServer.h

```diff
@@ -34,6 +34,10 @@
 #error "Please define the name of the COM process!!!"
 #endif
 
+#ifdef THUNDER_CRASH_HANDLER
+#define MAX_LENGTH_JSON_REQUEST_PARAM 200 /* maximum length of json request param to store */
+#endif
```

Comment:
Preprocessor statements are from the old days :-) 
The disadvantage of preprocessor statements are that the do not use namespacing and thus might conflict easilly. Why not use a proper const definition within the PluginServer class namespace ?